learn/classes/super_class.md

```diff
@@ -0,0 +1,263 @@
+In this article, we will delve into the intricacies of Python Super Classes. Get ready to unlock the full potential of Python Development by mastering the vital elements of Super Class in Python.  
+  
+## Understanding Python's Super Class  
+
+The `super()` function in python is used to call a method in a parent class from a child class. The `super()` function helps in creating a hierarchy among classes in Python that is easy to maintain and understand. In Python 3, `super()` is commonly used for initializing abstract classes.
+
+### Using `super()` in Python Class
+
+```python
+class MyParentClass:
+    def do_something(self):
+        print("Parent class doing something.")
+
+class MyChildClass(MyParentClass):
+    def do_something(self):
+        super().do_something()
+        print("Child class doing something.")
+        
+child = MyChildClass()
+child.do_something()
+```
+
+In conclusion, `super()` is a powerful function in Python that provides a clean and concise way to call parent class methods from a child class. It helps to create a clear hierarchy among classes and to maintain clean code.  
+  
+## Python 3 Abstract Class  
+
+Abstract class is a class that cannot be instantiated and is meant to be subclassed by other classes. The `super()` function is used to access and call methods from a parent or superclass in Python. Here are two code examples that illustrate the use of `super()` in Python 3 abstract classes:
+
+### Python 3 Abstract Class. Example 1
```

Comment:
Rewrite the heading. It's very similar to H2 + Example word is not the best keyword. Let's use something with 'super', 'init'.